Yoast/javascript
# Added error handling when building the tree in the YoastSEO.js devtool.

## Summary
* _[not-user-facing]_ Adds error handling to the YoastSEO development tool, when building the tree for visualization purposes.

## Relevant technical choices
N/A

## Test instructions
* Start the YoastSEO.js development tool by running `yarn start` in `javascript/apps/content-analysis`.
* In the development tool: make sure that the "Build the tree" feature toggle is set to "on".
* Enter a HTML-text that currently fails to build the tree.
   * For example:
     ```html
     <h1>This is a </h1>
     ``` 
     (See issue #38 )
* The example should **not** crash.
* The browser's development console **should** show an error.

Fixes #129
